v2/client/src/components/pages/TrainerFeedback/index.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Redirect } from 'react-router-dom';
+
+import Feedback from '../../common/Feedback';
+
+import { Wrapper, Paragraph } from './TrainerFeedback.style';
+import Header from '../../common/Header';
+
+// this can be refactored later on
+const urlArr = window.location.href.split('/');
```

Comment:
not sure what you are trying to do here 

you can get any attribute straight from react-router rather than this convoluted way

Reply:
probably need to change that weird trainer id / session id thing anyway so leave this for now